### PR TITLE
[PM-9921] Add Popout to Add/Edit/View components

### PR DESCRIPTION
**Pull request by nick-livefront**
## 🎟️ Tracking

[PM-9921](https://bitwarden.atlassian.net/browse/PM-9921)

## 📔 Objective

Add `PopOutComponent` to the header of the Add/Edit/View cipher components within the browser refresh

### Note

The back button behavior is broken once the extension is popped out. I expect that to be fixed in [PR 9556](https://github.com/bitwarden/clients/pull/9556) & [PM-9455](https://bitwarden.atlassian.net/browse/PM-9455)  

## 📸 Screenshots

https://github.com/user-attachments/assets/6a382a13-1bbf-4e13-867f-50c0830b1c49

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

[PM-9921]: https://bitwarden.atlassian.net/browse/PM-9921?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PM-9455]: https://bitwarden.atlassian.net/browse/PM-9455?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Jingo88 — DISMISSED** [review]

File: apps/browser/src/vault/popup/components/vault-v2/view-v2/view-v2.component.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <popup-page>
-  <popup-header slot="header" [pageTitle]="headerText" showBackButton> </popup-header>
+  <popup-header slot="header" [pageTitle]="headerText" showBackButton
+    ><app-pop-out slot="end"
+  /></popup-header>
```
Comment (shane-melton) [inline apps/browser/src/vault/popup/components/vault-v2/view-v2/view-v2.component.html:4]: 💭 Why is the HTML formatter so weird.. 😕

**Review by shane-melton — APPROVED** [review]
Looks good!

File: apps/browser/src/vault/popup/components/vault-v2/view-v2/view-v2.component.html (hunk 1 above)
Comment (nick-livefront) [reply apps/browser/src/vault/popup/components/vault-v2/view-v2/view-v2.component.html]: @shane-melton This was worth fixing, imo. [14bb11e](https://github.com/bitwarden/clients/pull/10382/commits/14bb11ea3e087125b89449d3207d56997516421e)

**Review by shane-melton — APPROVED** [review]
Thanks!

**Pull request merged**